tests/test-cases/Group6-VIC-Machine/6-13-TLS.robot

```diff
@@ -32,7 +32,27 @@ Create VCH - defaults with --no-tls
     Run Regression Tests
     Cleanup VIC Appliance On Test Server
 
+Create VCH - defaults custom cert path
+    Set Test Environment Variables
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+
+    ${pwd}=     Run  pwd
+    ${output}=  Run  bin/vic-machine-linux create ${vicmachinetls} --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} --bridge-network=%{BRIDGE_NETWORK} --public-network=%{PUBLIC_NETWORK} --cert-path=${pwd}/foo-bar-certs/
+    Should Contain  ${output}  --tlscacert="${pwd}/foo-bar-certs/ca.pem" --tlscert="${pwd}/foo-bar-certs/cert.pem" --tlskey="${pwd}/foo-bar-certs/key.pem"
```

Comment:
yeah this works. I had accidentally put `$(pwd)` on my first iteration and that didn't pass, but this does, which indicates to me that that argument is interpreted as a string